howto/monitor/collect-metrics.md

```diff
@@ -0,0 +1,227 @@
+This example implementation provides a starting point for a monitoring stack that can be used to gather metrics with [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) and [Prometheus](https://prometheus.io/) and access them through [Grafana](https://grafana.com/).
+
+In this setup, every LXD cluster node runs a Telegraf instance that gathers the machine metrics. All Anbox containers that exist on the node also report their metrics to the Telegraf instance.
+
+Prometheus gathers the metrics that are collected by the different Telegraf instances and combines them with the metrics provided directly by AMS. You can access, query and visualise the full metrics data through Grafana.
+
+![Architecture for collecting metrics](../../images/collect-metrics.png)
+
+## Deploy the solution
+
+The monitoring stack is provided as an [overlay file](https://discourse.ubuntu.com/t/installation-customizing/17747#overlay-files) that you use when deploying Anbox Cloud.
+
+The overlay file adds a machine to the deployment (with ID `2` for the `anbox-cloud-core` bundle or ID `3` for the `anbox-cloud` bundle), which is used to run Grafana and Prometheus. In addition, the overlay installs and configures Telegraf for both AMS and the LXD nodes, and it creates the required relations.
+
+The ideal requirements for the additional machine are as follows:
```

Comment:
Lets call them "minimum" requirements. You will want to to allocate more than this for a production environment but shouldn't start here to describe what to consider for that.